core/storage/user/gae_models_test.py

```diff
@@ -44,6 +44,51 @@ def test_get_by_role(self):
             feconf.ROLE_ID_ADMIN)
         self.assertEqual(user[0].role, feconf.ROLE_ID_ADMIN)
 
+    def test_export_data(self):
+        user = user_models.UserSettingsModel.get_by_role(
+            feconf.ROLE_ID_ADMIN)[0]
+        user_data = user.export_data(user.id)
+        self.assertEqual(user.email, user_data['email'])
```

Comment:
Use the actual field values rather than user.email, etc.

That will help us get some sense of any cases where we're only testing trivial things (e.g. if there are links to other model IDs in this model but we're only testing the empty-list case) and suggest areas where we should improve the test coverage.

Reply:
So to clarify, I currently have this testing None, default values, and empty lists, is this what we want?

Reply:
Yes!